generators/extensions/dx-asset-compute-worker-1/index.js

```diff
@@ -60,7 +60,7 @@ class DxAssetComputeWorker1 extends Generator {
       'extensions.' + this.configName,
       // value
       {
-        $include: this.extConfigPath
+        $include: this.extConfigPath.split(path.sep).join(path.posix.sep)
```

Comment:
there is a module in the installed dependencies for converting to unix paths: https://github.com/adobe/generator-aio-app/blob/bug/ACNA-1108-windows-bootstrapped-fail-unix/package.json#L52 using  `upath.toUnix()`
I think it is better to use that one, as there might be some corner cases in the future with absolut paths like `C:\a\b\c`